loops.js

```diff
@@ -87,5 +109,19 @@
 // etc.
 // ==========================================
 
-
+for(let i = 1; i <= 100; i++) {
+    if(i%3===0 && i%5===0) {
+        console.log("FizzBuzz");
+    }
+
+    else if(i%3===0){
+        console.log("Fizz");
+    }
+    else if(i%5===0){
+        console.log("Buzz");
+    }
+    else{
+        console.log(i);
+    }
+}
 
```

Comment:
De FizzBuzz-oplossing is correct en overzichtelijk. De volgorde van de voorwaarden is juist, waardoor de juiste woorden worden geprint.